modules/nf-core/fastqc/main.nf

```diff
@@ -21,9 +21,12 @@ process FASTQC {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def rename_to = reads instanceof Path || reads.size() == 1 ? "$reads ${prefix}.${reads.extension}" : reads.withIndex().collect { entry, index -> "$entry ${prefix}_${index + 1}.${entry.extension}" }.join(' ')
     """
-    printf "%s\\n" $reads | while read f; do [[ \$f =~ ^${prefix}.* ]] || ln -s \$f ${prefix}_\$f ; done
-    fastqc $args --threads $task.cpus ${prefix}*
+    printf "%s %s\\n" $rename_to | while read old_name new_name; do
+        [ -f "\${new_name}" ] || ln -s \$old_name \$new_name
+    done
+    fastqc $args --threads $task.cpus ${rename_to.tokenize().withIndex().findAll{ entry, index -> index % 2 }.collect{ it[0] }.join(' ')}
```

Comment:
Well, this took a turn for the worse! 🤣

Reply:
don't I know it... Got a better suggestion.

Reply:
I don't I'm afraid. I did try playing with it on Friday but it turned into a can of worms when trying to deal with PE reads. This is why I suggested going back to the previous implementation but having another input value for interleaved.